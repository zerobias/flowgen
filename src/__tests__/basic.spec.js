import { compiler, beautify } from "..";

it("should handle basic keywords", () => {
  const ts = `type A = {
    a: void,
    b: string,
    c: any,
    d: number,
    e: boolean,
    f: null,
    g: undefined,
    h: object,
    i: 1,
    j: 2,
  }`;
  const result = compiler.compileDefinitionString(ts);
  expect(beautify(result)).toMatchSnapshot();
});
