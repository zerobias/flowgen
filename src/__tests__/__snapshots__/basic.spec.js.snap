// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle basic keywords 1`] = `
"declare type A = {
  a: void,
  b: string,
  c: any,
  d: number,
  e: boolean,
  f: null,
  g: void,
  h: { [key: string]: any },
  i: 1,
  j: 2
};
"
`;
